gitgud/test_operator.py

```diff
@@ -0,0 +1,54 @@
+import pytest
+
+from gitgud.skills import all_skills
+from gitgud.operations import get_operator
+
+
+@pytest.fixture
+def file_operator(gg):
```

Comment:
I don't think you need gg as a parameter here. Did we already talk about this?

Reply:
No, not that I recall. This `gg` is here because I wanted to assure that Git Gud was initialized since `file_operator` would only ever be `None` if Git Gud was intialized.

Reply:
That's fine, feel free to leave as is.